vnext/src/Libraries/Components/Picker/PickerWindows.tsx

```diff
@@ -85,20 +85,19 @@ export class Picker extends React.Component<IPickerProps, State> {
   }
 
   public render(): JSX.Element {
-    const props = {...this.props};
-    props.onStartShouldSetResponder = () => true;
-    props.onResponderTerminationRequest = () => false;
-    props.style = this.props.style;
```

Comment:
This line doesn't seem to do anything useful. We already do a shallow clone of props including `props.style` with `const props = {...this.props};` above.